build/constants.js

```diff
@@ -32,6 +32,15 @@ const FOLDERSEARCH = process.env.BUILD_FOLDERSEARCH || "";
 const NO_PROGRESSBAR = Boolean(
   JSON.parse(process.env.BUILD_NO_PROGRESSBAR || process.env.CI || "false")
 );
+const FIX_FLAWS = JSON.parse(process.env.BUILD_FIX_FLAWS || "false");
+const FIX_FLAWS_DRY_RUN = JSON.parse(
+  process.env.BUILD_FIX_FLAWS_DRY_RUN || "false"
+);
+const FIX_FLAWS_VERBOSE = JSON.parse(
```

Comment:
At the moment, I can't think of a reason not to have it print a bunch of stuff to stdout when it does stuff. But what's done is done for now. It's not a horrible thing to have.